### ISSUE TITLE
pprof default port question

### ISSUE THREAD

**Issue by scottschulthess** [cloudfoundry/loggregator-release#368]
Hey everyone, previously I've been able to get a pprof dump easily because pprof was set up on port 6060 in loggregator somewhat recently it changed to 0 https://github.com/cloudfoundry/loggregator-release/blob/develop/jobs/doppler/spec#L58.  in this commit https://github.com/cloudfoundry/loggregator-release/commit/b62e8eee3fd64e48ee240e864a9e7c736331cff1  Is it correct that in newer versions operators need to redeploy to enable this?   cc @wfernandes (sorry for some somewhat obvious questions)

Also https://github.com/cloudfoundry/loggregator/blob/master/profiler/server.go doesn't appear to check to see if the port is set to >0 before starting, so I'm curious how that works when it is set to 0? so if pprof it is indeed disabled by default I'm curious why and how that works with my reading of the profiler code in loggregator.

**Comment by bradylove** [comment]
@scottschulthess we changed the port to 0 to cause the pprof server to start on a random port. Setting to 0 does not disable the pprof server.

Since we start on a random port you would need to use `lsof` to find the pprof port in order to use it. An example of how to do this would be, as sudo, `lsof -c doppler | grep LISTEN`.

**Comment by scottschulthess** [comment]
@bradylove that's awsome, thank you for the quick explanation! 
 should I file a separate issue to update https://github.com/cloudfoundry/loggregator-release/wiki/FAQ

**Comment by bradylove** [comment]
@scottschulthess that would be great, thank you.
